fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/services/RepositoryServiceImpl.java

```diff
@@ -97,14 +99,20 @@ public Long getRepositoryObjectCount() {
      * .Session, java.io.File)
      */
     @Override
-    public Problems backupRepository(final Session session,
+    public Collection<Throwable> backupRepository(final Session session,
                                      final File backupDirectory) {
         try {
             final RepositoryManager repoMgr = ((org.modeshape.jcr.api.Session) session)
                     .getWorkspace()
                     .getRepositoryManager();
 
-            return repoMgr.backupRepository(backupDirectory);
+            final Collection<Throwable> problems = new ArrayList<>();
+
+            repoMgr.backupRepository(backupDirectory).forEach(x -> {
```

Comment:
No need for braces and newlines here, just `.backupRepository(backupDirectory).forEach(x -> problems.add(x.getThrowable()));`